test/apiv2/10-images.at

```diff
@@ -239,4 +239,18 @@ fi
 
 cleanBuildTest
 
+# compat API delete image produces a delete event
```

Comment:
Suggestion: make the difference clearer, e.g., `...delete event; libpod API calls it 'remove'`